.circleci/generate.sh

```diff
@@ -332,8 +336,72 @@ delete_repeated_jobs()
   fi
 }
 
+# Define function to leave only a single config run for each test group
+# The first and only argument is the file name.
```

Comment:
Maybe we mention in the comment also that this will be used during dev, not pre-commit?